js/main.js

```diff
@@ -28,9 +32,53 @@ $entryForm.addEventListener('submit', function (event) {
     entryNotes: $entryNotes.value,
     entryId: data.nextEntryId++
   };
-  data.entries.unshift(entryFormObject);
+  data.entries.push(entryFormObject);
   $entryImage.setAttribute('src', 'images/placeholder-image-square.jpg');
   $entryImage.setAttribute('alt', 'Placeholder Image');
   $entryImage.setAttribute('title', 'Placeholder Image');
   $entryForm.firstElementChild.reset();
+  setEntry(entryFormObject);
+});
+
+function setEntry(dataEntry) {
+  var $newEntry = document.createElement('li');
+  $newEntry.className = 'row';
+  $newEntry.setAttribute('id', dataEntry.entryId);
+  var $leftColumn = document.createElement('div');
+  $leftColumn.className = 'column-half left';
+  $newEntry.appendChild($leftColumn);
+  var $leftImage = document.createElement('img');
+  $leftImage.setAttribute('src', dataEntry.entryUrl);
+  $leftColumn.appendChild($leftImage);
+  var $rightColumn = document.createElement('div');
+  $rightColumn.className = 'column-half right';
+  $newEntry.appendChild($rightColumn);
+  var $rightTitle = document.createElement('h2');
+  $rightTitle.textContent = dataEntry.entryTitle;
+  $rightColumn.appendChild($rightTitle);
+  var $rightNotes = document.createElement('p');
+  $rightNotes.textContent = dataEntry.entryNotes;
+  $rightColumn.appendChild($rightNotes);
+  $ulEntries.prepend($newEntry);
+  entryList['entryId #' + dataEntry.entryId] = $newEntry;
```

Comment:
What is the purpose of this `entryList` variable? You already have `data.entries`. Property names should also never include spaces or special characters, they should follow the same rules as variable names.

Reply:
good to know! This one I made to hold the DOM object for each entry, though I suppose I can get them by their `id` number later on